Dockerfile

```diff
@@ -3,5 +3,5 @@ ENV LANG=en_US.UTF-8
 USER 0
 RUN yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
```

Comment:
The URL seems to be same based on https://www.postgresql.org/download/linux/redhat/

Reply:
I believe its the same too. :)